Makefile

```diff
@@ -0,0 +1,14 @@
+clean:
+	rm -rf ./dist
+
+build-erdpy: clean
+	python3 setup.py sdist
+
+publish-erdpy: build-erdpy
+	twine upload dist/*
+
+arwendebug:
```

Comment:
Not necessary anymore (it was for erdtest-js).

Reply:
awesome. removed